examples/dynamic/mne_var_connectivity.py

```diff
@@ -0,0 +1,225 @@
+"""
+.. _ex-var:
+
+===================================================
+Compute vector autoregressive model (linear system)
+===================================================
+
+Compute a VAR (linear system) model from time-series
+activity :footcite:`li_linear_2017` using a
+continuous iEEG recording.
+
+In this example, we will demonstrate how to compute
+a VAR model with different statistical assumptions.
+"""
+
+# Authors: Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
+import numpy as np
+
+import mne
+from mne import make_fixed_length_epochs
+from mne_bids import BIDSPath, read_raw_bids
+
+import matplotlib.pyplot as plt
+
+from mne_connectivity import vector_auto_regression
+
+# %%
+# Load the data
+# -------------
+# Here, we first download an ECoG dataset that was
+# recorded from a patient with epilepsy. To facilitate
+# loading the data, we use
```

Comment:
nitpick but I would suggest to fill as much as possible 80c per line in your comments. Here and below

Reply:
Filled in more lines.